worker/categorize.py

```diff
@@ -381,6 +381,7 @@ def _get_best_keywords(self, communities, original_keywords):
     def _is_viable_candidate(self, phrase):
 
         # Ignore 1-character phrases
+        phrase = phrase.strip()
```

Comment:
Actually, this isn't good because words with spaces will be stored into the category db with their spaces. Let's take this strip and move it before _is_viable_candidate is even called to make our categories cleaner. Wikipedia would also have issues with words with spaces at either end.